packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -68,6 +68,11 @@ typedef SemanticsUpdateCallback = void Function(ui.SemanticsUpdate update);
 /// value.
 typedef ChildSemanticsConfigurationsDelegate = ChildSemanticsConfigurationsResult Function(List<SemanticsConfiguration>);
 
+final int _kSystemInteractions = SemanticsAction.didGainAccessibilityFocus.index
+  | SemanticsAction.didLoseAccessibilityFocus.index
+  | SemanticsAction.customAction.index;
```

Comment:
I think custom actions are usually non pointer related action

Reply:
I forget how custom actions are exposed in TalkBack and VoiceOver. How do users trigger these? It seems odd that they are still allowed, though? Would probably be good to look at some use cases of custom action and decide based on that whether it would make more sense to allow or disallow these. (If it is unclear whether they should be blocked in all cases maybe we need to make it configurable after all which actions are blocked, at least on the SemanticsConfiguration level).

Reply:
Actually, I thought these expose actions that non-a11y-users would usually trigger via pointers. For example to delete an entry in a list, etc.

Reply:
This is the only remaining open question on the PR. For me, if I have an interface where I can delete a ListItem by swiping right/left on it and I expose this action as a custom action for a11y users, I would expect it gets blocked when I use an IgnorePointer. Everything else would be surprising. I can't think of a use case where I would still want custom actions to be allowed. Do you have one? If so, we may need to make this configurable if it can go both ways.